pkg/preparer/orchestrate_test.go

```diff
@@ -82,7 +84,9 @@ func TestPreparerLaunchesPodsThatHaveDifferentSHAs(t *testing.T) {
 		currentManifest: existing,
 	}
 	newManifest := testManifest(t)
-	success := installAndLaunchPod(newManifest, testPod, logging.DefaultLogger)
+
+	var p *Preparer
```

Comment:
Ok I understand - it just happens to work because the function is on `*Preparer` not `Preparer`, and nothing is dereferenced about that pointer specifically. Seems like a timebomb for tests.